code/modules/clothing/under/jobs/engineering.dm

```diff
@@ -1,3 +1,10 @@
+//Fits all uniforms to Vox
+/obj/item/clothing/under/rank/

```

Comment:
this shouldn't be defined multiple times; likewise, making one for clothing/under will catch clothing/under/rank.